v5/Http/Resources/PostResource.php

```diff
@@ -31,31 +33,32 @@ private function includeResourceFields($request)
             'created',
             'updated',
             'post_date',
-//            'base_language' => $this->base_language,
-//            'translations' => new TranslationCollection($this->translations),
-//            'enabled_languages' => [
-//                'default'=> $this->base_language,
-//                'available' => $this->translations->groupBy('language')->keys()
-//            ],
+            //            'base_language' => $this->base_language,
+            //            'translations' => new TranslationCollection($this->translations),
+            //            'enabled_languages' => [
+            //                'default'=> $this->base_language,
+            //                'available' => $this->translations->groupBy('language')->keys()
+            //            ],
         ]);
     }
+
     private function getResourcePostContent()
     {
-
-        $values = $this->getPostValues();
-        $col = new Collection(['values' => $values, 'tasks' => $this->survey ? $this->survey->tasks : []]);
-        $no_values = false;
-
-        if ($values->count() === 0) {
-            $no_values = true;
-        }
-        $post_content =  Collection::make([]);
+        $values = $this->getPostValues(); // Calling method on Post Model
+        $no_values = $values->count() === 0 ? true : false;
```

Comment:
Just curious, was this line written by hand or auto-generated?

Reply:
It was written by me, needed to make that code block eloquent.